ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -155,20 +158,18 @@ public class BottomSheetController: UIViewController {
     ///   - animated: Indicates if the change should be animated. The default value is `true`.
     ///   - completion: Closure to be called when the state change completes.
     @objc public func setIsExpanded(_ isExpanded: Bool, animated: Bool = true, completion: ((_ isFinished: Bool) -> Void)? = nil) {
-        guard isExpanded != self.isExpanded else {
+        guard isExpanded != self.isExpanded, !isHiddenOrHiding, isExpandable else {
```

Comment:
question out of the scope of this change:

I noticed that both setIsExpanded and setIsHidden take the same set of params:
```
setIsExpanded(_ isExpanded: Bool, animated: Bool = true, completion: ((_ isFinished: Bool) -> Void)? = nil)
setIsHidden(_ isHidden: Bool, animated: Bool = true, completion: ((_ isFinished: Bool) -> Void)? = nil)
```

Is it possible to programmatically dismiss  the bottom sheet (hide by moving it out of the screen) when it's  in the expanded state?

The suggestion that crossed my mind is to consolidated these 2 funcs (at least in the public API) in one so we can remove one public API and still provide the dismissal from expanded directly (without calling setIsExpanded(false) and then setIsHidden(true)).

Thoughts?

Reply:
Calling setIsHidden(true) will override isExpanded and hide the sheet regardless of its current expansion state.
The main issue is that isExpanded is not always the opposite of isHidden. When isExpanded is false, the sheet could either be visible and collapsed, or hidden.

Reply:
(Resolving to unblock PR, but feel free to continue discussing here.)

Reply:
My suggestion on incorporating both APIs into one while still keeping the same concept of hidden, collapsed and extended states. 

For that the parameter would need to be the **BottomSheetExpansionState** enumeration instead of the boolean.
From a caller standpoint that could be easier to understand.

Thoughts?